server/tests/helpers/date_helper.js

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
I followed the convention of the helper that already exists, so if I change this one I'll change the other two. 

We don't have a convention for all files, sometimes they're snake and sometimes camelCased, so which do we want to go with?